package.json

```diff
@@ -86,7 +87,9 @@
     "version": "yarn run version-checkout && git add package.json && yarn run version-commit",
     "postversion": "git push --set-upstream ${GIT_REMOTE:-origin} release/v$npm_package_version",
     "precreate-release-pr": "npx swc ./scripts/create-release-pr.ts -o ./scripts/create-release-pr.mjs",
-    "create-release-pr": "node ./scripts/create-release-pr.mjs"
+    "create-release-pr": "node ./scripts/create-release-pr.mjs",
+    "prefix-canvas-deps": "npx swc ./scripts/fix-canvas-deps.ts -o ./scripts/fix-canvas-deps.mjs",
+    "fix-canvas-deps": "node ./scripts/fix-canvas-deps.mjs"
```

Comment:
What is the requirement for `canvas` in the first place? This is complexity which shouldn't be included without very good reason.

Reply:
Canvas is required for some of the tests IIRC, might not be necessary if we upgrade to xtermJs 5 which adds the DOM based renderer for testing

Reply:
With this complexity, I would say that we stop testing xtermjs directly and only test the contract (our wrapper).